src/client/app/access-checker/admin-cache-entries.component.ts

```diff
@@ -38,8 +38,8 @@ export class AdminCacheEntriesComponent {
 		this.alertService.clearAllAlerts();
 
 		this.pagingOpts = new PagingOptions();
-		this.pagingOpts.sortField = this.sortOpts['key'].sortField;
-		this.pagingOpts.sortDir = this.sortOpts['key'].sortDir;
+		this.pagingOpts.sortField = this.sortOpts.key.sortField;
```

Comment:
Then the linting rules need to be changed; these lines were causing my PRs to fail the tests, and only by fixing them was I able to get it to succeed.